docs/userguide.rst

```diff
@@ -315,6 +315,33 @@ and push the created image to the DockerHub image registry:
 
     $ docker push johndoe/myenv
 
+**Supporting non-root users**
+
+Running containers with root user is not secure and therefore dicouraged. If
+you want to improve the security in your environment you can setup your own
+user.
+
+In order to be compatible with the REANA platform you should do the following:
+
+- Add your user to the root group, so anything created by this user will be
+  accessible by the REANA user, who also belongs to the root group.
+
+- Any directory or files used in your analysis should have the correct
+  permissions, so the REANA user can access them.
+
+If we put everything together, in orther to create an environment with
+previous ``Dockerfile`` supporting a custom user, we would only need to add
+the following to the end of the file:
```

Comment:
I would rephrase for example to ``the following code to the end...`` or ``the following commands to the end...``